docs/guides/getting-started-java-spring.md

```diff
@@ -82,22 +82,23 @@ Next, we'll create a BPMN diagram to represent the transaction model shown at th
    2. **Script/Result variable**: `totalWithTax`
    3. **Script/FEEL expression**: `total * 1.1` (this represents the tax applied to the transaction.)
 7. Append a task named `Charge credit card`.
-8. Click on the task and click the wrench-shaped icon to change the type of task to a service task. In the properties panel, change the **Task definition/Type** to `charge-credit-card`.
+8. Click on the task and click the wrench-shaped icon to change the type of task to a service task. In the properties panel, change the **Task definition/Job type** to `charge-credit-card`.
```

Comment:
This description probably changed since we wrote the guide, it now reads "Job type" in the UI of Desktop Modeler.